scripts/createPrivateKey.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Create a local random private key and writing it and the mneumonic to your .env file.
+ * This is designed to accelerate demos and getting started with this repo.
+ */
+const bip39 = require('bip39');
+const ethers = require('ethers');
+const fs = require('fs');
+const path = require('path');
+
+
+if (!fs.existsSync('.env')) {
+    const contents = fs.readFileSync(__dirname + '/../.env.example');
+    console.log('Generating new random key.');
+    // Generate mneumonic
+    const mnemonic = bip39.generateMnemonic();
+    // Convert to private key
+    const wallet = ethers.Wallet.fromMnemonic(mnemonic);
+
+    // Write to .env in the root of the project
+    console.log('Creating .env file in ', path.dirname(__dirname));
+    const newEnvFile = contents.toString().replace('YOUR_PRIVATE_KEY_HERE',`${wallet.privateKey}\nMNEUMONIC=${mnemonic}`);
```

Comment:
i know it's not ready for review yet, but I just come to your pr and spot this typo.

Reply:
rebased and fixed `mneumonic` to be `mnemonic`